schema.graphql

```diff
@@ -121,3 +121,17 @@ type NativeTransfer @entity {
   transaction: Transaction!
   block: Block!
 }
+
+type Account @entity {
+  # id is the address
+  id: ID!
+  nativeBalances: [NativeBalance]! @derivedFrom(field: "account")
+}
+
+type NativeBalance @entity {
```

Comment:
@daeMOn63 what do you think about renaming this to `NativeBalanceChange`? I find the current name confusing given the direction in which we've iterated the design.